packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -0,0 +1,98 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Button: ComponentStyleConfig = {
+  baseStyle: {
+    fontWeight: 'medium',
+    borderRadius: 'base', // <-- border radius is same for all variants and sizes
+    color: 'text-light',
+  },
+  // Two sizes: sm and md
+  sizes: {
+    xs: {
+      fontSize: 12,
+      lineHeight: 4,
+      letterSpacing: 'wide',
+      px: 2,
+      py: 2.5,
+    },
+    sm: {
+      fontSize: 14,
+      lineHeight: 5,
+      px: 3,
+      py: 2.5,
+    },
+    md: {
+      fontSize: 16,
+      lineHeight: 6,
+      px: 4,
+      py: 2.5,
+    },
+    lg: {
+      fontSize: 18,
+      lineHeight: 7,
+      px: 6,
+      py: 2.5,
+    },
+  },
```

Comment:
this has to be done using the styles defined in `src/theme/ComposerTheme/styles.ts` or the `Text` component